liquidctl/driver/hydro_platinum.py

```diff
@@ -164,6 +164,17 @@ def initialize(self, pump_mode='balanced', **kwargs):
         Returns a list of `(property, value, unit)` tuples.
         """
 
+        # These hex strings are currently magic values that work but Im not quite sure why.
```

Comment:
If these aren't needed unless to set LED settings, then they should only be sent in that API.

---

Additionally, I try to avoid sending opaque blobs: even when they fix a current issue, they are unmaintainable.

I'm also worried these in particular could be storing non-volatile state, which together with the opaqueness, could lead us to brick devices: if these do anything to non-volatile state we need to know (at least to some degree) their contract.  For example: if `d2` fails, do we just abort `d3` and (later) restart at `d1` or do we need some unknown `d4` to clear it up?

Again, if these weren't so opaque, it would be easier to reason about.  So, practically, can you try to distill these down to the underlying byte and/or bit flags that are being sent?

Like, are these always constant? Do any of these change when you toggle/vary different settings on iCue?  When are these sent exactly, and is it always the three, in this sequence, and with nothing else in between?  What does the cooler respond to each one, and are the responses the same too? Etc.

Reply:
> Like, are these always constant?

From the different wireshark captures that I have they are the almost the exact same every time they are sent. The only bytes that I have noticed are different is the first 2 `0101` is sometimes `0000`, but I have not been able to tell why. I will look into it a little more tho. 


> If these aren't needed unless to set LED settings, then they should only be sent in that API.

Thats a good point, I can move this to the `set_color()` and set a persistent value to not send it again if it has already been set.